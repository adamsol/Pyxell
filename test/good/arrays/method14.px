
a = [-1, 4, 6]
print a.has(4), a.has(-1), a.has(6)
print a.has(1), a.has(46)

print [[]].has([])
print [1f..10^6].has(10^^5)
print [15e-1..10^6].has(10^5)
