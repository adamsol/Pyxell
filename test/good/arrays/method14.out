true true true
false false
true
true
false
